<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorDebugDialog</class>
 <widget class="QWidget" name="MotorDebugDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>电机调试</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>电机名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>边缘电机3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>边缘电机4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>传动带电机</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="posEdit3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="posEdit4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="posEditBelt">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>当前位置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>速度设定</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxBelt">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="upBtn3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>向上</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upBtn4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>向上</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upBtnBelt">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>前进</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downBtnBelt">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>150</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>后退</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downBtn4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>向下</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downBtn3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>向下</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopBtnBelt">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>150</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopBtn3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>70</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopBtn4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="manualCheckBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>手动模式</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
